src/main/java/uk/gov/nationalarchives/pdi/step/xml/CanonicalStepMeta.java

```diff
@@ -0,0 +1,217 @@
+package uk.gov.nationalarchives.pdi.step.xml;
+
+import org.eclipse.swt.widgets.Shell;
+import org.pentaho.di.core.annotations.Step;
+import org.pentaho.di.core.database.DatabaseMeta;
+import org.pentaho.di.core.exception.KettleException;
+import org.pentaho.di.core.exception.KettleStepException;
+import org.pentaho.di.core.exception.KettleValueException;
+import org.pentaho.di.core.exception.KettleXMLException;
+import org.pentaho.di.core.row.RowMetaInterface;
+import org.pentaho.di.core.row.ValueMetaInterface;
+import org.pentaho.di.core.row.value.ValueMetaString;
+import org.pentaho.di.core.variables.VariableSpace;
+import org.pentaho.di.core.xml.XMLHandler;
+import org.pentaho.di.repository.ObjectId;
+import org.pentaho.di.repository.Repository;
+import org.pentaho.di.trans.Trans;
+import org.pentaho.di.trans.TransMeta;
+import org.pentaho.di.trans.step.*;
+import org.pentaho.metastore.api.IMetaStore;
+import org.w3c.dom.Node;
+
+import java.util.List;
+
+@Step(
+        id = "CanonicalStep",
+        name = "CanonicalStep.Name",
+        description = "CanonicalStep.TooltipDesc",
+        image = "CanonicalStep.svg",
+        categoryDescription = "i18n:org.pentaho.di.trans.step:BaseStep.Category.Transform",
```

Comment:
Ooooh! The internationalisation of the `categoryDescription` here is a nice touch. Would you be willing to send PRs to kettle-jena-plugins and kettle-atomic-plugins to do the same?

Reply:
Mmmm :-) I see most of these are internationalised - so if you could include that in the PRs too please?